### PR TITLE
Limit node CPUs on VSphere

### PR DESCRIPTION
**Pull request by kgroschoff**
**What this PR does / why we need it**:
Limit the amount of CPUs per node to a maximum of 8 on VSphere.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #https://github.com/kubermatic/kubermatic/issues/2139

**Special notes for your reviewer**:
/

```release-note
Limit amount of CPUs per node to a maximum of 8 on VSphere
```

### CODE REVIEW

File: pkg/cloudprovider/provider/vsphere/provider.go [hunk 1]
```diff
@@ -277,6 +277,10 @@ func (p *provider) Validate(spec v1alpha1.MachineSpec) error {
 		return errors.New("specified target network (VMNetName) in cluster, but no source network (TemplateNetName) in machine")
 	}
 
+	if config.CPUs > 8 {
+		return errors.New("number of CPUs could not be greater than 8")
```
Comment (alvaroaleman) [inline pkg/cloudprovider/provider/vsphere/provider.go]: can you replace `could` with `must not`? Other than that LGTM

Comment (kgroschoff) [reply pkg/cloudprovider/provider/vsphere/provider.go]: 👍

**Pull request merged**